.github/workflows/build_and_test.yml

```diff
@@ -870,7 +869,7 @@ jobs:
           fi
 
       - name: Verify docker-compose example and test extensions
-        timeout-minutes: 20
+        timeout-minutes: 40
```

Comment:
I think we need to parallelise this job instead

Reply:
If we decide to test both versions, yes.

Reply:
done